panel/chat/langchain.py

```diff
@@ -9,7 +9,7 @@
     AgentFinish = None
```

Comment:
You also need to update the links to the reference guides in the docstrings. For example https://panel.holoviz.org/reference/widgets/ChatInterface.html

Reply:
Thanks for catching that.